src/mainboard/opentitan/crb/.cargo/config

```diff
@@ -0,0 +1,7 @@
+[build]
+target = "riscv32imc-unknown-none-elf"
```

Comment:
technically, this ought to work in the cpu Cargo.toml; can you try it?

Reply:
No, it doesn't seem to work. It appears to use the .cargo at the topmost layer.